components/comment/Form.js

```diff
@@ -0,0 +1,33 @@
+export default function Form({ onSubmit }) {
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+
+    onSubmit(data.tag);
+
+    event.target.reset();
+    event.target.elements.tag.focus();
+  }
+
+  return (
+    <form className="form" onSubmit={handleSubmit}>
```

Comment:
we don't work with classNames but with styled components.
See our session handout [as reference](https://github.com/neuefische/web-bo-22-2/blob/main/sessions/react-styled-components/react-styled-components.md).